controllers/snapshot/snapshot_adapter.go

```diff
@@ -237,7 +242,7 @@ TestScenarioLoop:
 
 		if err != nil {
 			a.logger.Error(err, "Copying of environment failed")
-			return reconciler.RequeueWithError(err)
+			return reconciler.RequeueAfter(requeueDelay, err)
```

Comment:
Why do we need manual requeueDelay, when we have by default exponential backoff retries when error happen in RequeueWithError.

Reply:
Removed the manual delay since we can't implement it currently due to the issue in https://github.com/redhat-appstudio/operator-goodies/blob/main/reconciler/handler.go#L27.